scripts/warnings.sh

```
@@ -0,0 +1,60 @@
+		run_ clang "$i" "${c_flags[@]}" -flto=full -fuse-ld=lld -Weverything  \
+		           "${sources[@]}" "$@"
+		run_ gcc   "$i" "${c_flags[@]}" -flto=auto \
+		           "${sources[@]}" "$@"
+	done
+
+}
+
+pushd "${0%/*}/" >/dev/null && {
+	pushd .. >/dev/null && {
+		run | sort -V -u
+		popd >/dev/null
```

Shellcheck (0.10.0)

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)